var/spack/repos/builtin/packages/slepc/package.py

```diff
@@ -83,6 +84,11 @@ def install(self, spec, prefix):
                     '--with-arpack-flags=-lparpack,-larpack'
                 ])
 
+        # It isn't possible to install BLOPEX separately and link to it;
+        # BLOPEX has to be downloaded with SLEPc at configure time
+        if '+blopex' in spec:
+            options.extend(['--download-blopex'])
```

Comment:
Can you replace `extend([])` with `append()`? No use creating a new list when you're only adding a single element.

Reply:
Sure.

Reply:
@adamjstewart Done.